bedrock/firefox/templates/firefox/features/picture-in-picture.html

```diff
@@ -0,0 +1,112 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ ftl('features-pip-firefox-picture-in-picture-get-more') }}{% endblock %}
+{% block page_desc %}{{ ftl('features-pip-got-things-to-do-and-things-to-watch') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/features/pip/pip-hero.jpg') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox-features-pip') }}
+{% endblock %}
+
+{% block sub_navigation %}
+  {% include "firefox/features/includes/subnav.html" %}
+{% endblock %}
+
+{% block content %}
+<section class="mzp-c-hero mzp-has-image mzp-t-dark">
+  <div class="mzp-l-content">
+    <div class="mzp-c-hero-body">
+      <h1 class="c-hero-pretitle"> {{ ftl('features-pip-cool-feature-picture-in-picture') }}</h1>
+      <h2 class="mzp-c-hero-title">{{ ftl('features-pip-get-more-done-with-pop-out-videos') }}</h2>
+      <div class="mzp-c-hero-desc">
+        <p>{{ ftl('features-pip-got-things-to-do-do-both-with') }}</p>
+      </div>
+      <div class="mzp-c-hero-cta">
+        <div class="download-firefox">
+          {{ download_firefox(dom_id='features-pip-download-primary', download_location='primary cta') }}
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="mzp-c-hero-image">
+    <img src="{{ static('img/firefox/features/pip/pip-hero.jpg') }}" alt="">
+  </div>
+</section>
+
+<div class="mzp-l-content mzp-t-content-xl">
+  <section class="mzp-c-card-feature mzp-has-aspect-1-1 mzp-l-card-feature-right-half mzp-t-firefox">
+    <div class="mzp-c-card-feature-media-wrapper">
+      <div class="mzp-c-card-feature-media">
+        <div class="video-container">
+          <div class="video-content">
+          {# YouTube is blocked in China so zh-CN gets an alternative, self-hosted video. #}
+          {% if LANG == 'zh-CN' %}
+            <video preload="metadata" controls poster="{{ static('img/firefox/features/pip/video-poster.jpg') }}" data-video-title="Red Panda Cubs - Firefox + Woodland Park Zoo">
+              <source src="https://assets.mozilla.net/video/red-pandas.webm" type="video/webm">
+            </video>
+          {% else %}
+            <a class="video-play js-video-play" href="https://youtu.be/F-nFQryDB0s" data-id="F-nFQryDB0s" data-video-title="Red Panda Cubs - Firefox + Woodland Park Zoo" title="{{ ftl('features-pip-play-the-video') }}">
+              <img src="{{ static('img/firefox/features/pip/video-poster.jpg') }}" alt="">
+            </a>
```

Comment:
When testing mobile OS'es (Android via BrowserStack and iOS vis xcode Device Simulator), when I clicked the video to play, I was prompted to download a `firefox.exe` file. This may be a simulator issue, but since it showed up in both seperate simluators, it may be a real thing. 

<img width="586" alt="image" src="https://user-images.githubusercontent.com/2692333/109689384-d964f280-7b4a-11eb-818b-3559a7e5243e.png">

<img width="911" alt="image" src="https://user-images.githubusercontent.com/2692333/109689528-fe596580-7b4a-11eb-8d06-4d0ce9dfa4c9.png">

Reply:
hmm... that doesn't happen when I test on my phone (Android), and I don't have an iOS device to test with. I'm guessing it's a simulator issue.

Reply:
I'll test on some native devices real quick to confirm. I think you're right.

Reply:
I also don't get the download on browserstack, though I was testing on demo4 not localhost, not sure if that makes a difference.

Reply:
Okay. Tested natively. Marking as resolved. 🛳️